css/my_portfolio.css

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
```

Comment:
- [ ] You've paid very good attention to the other requirements 👍. That's a job well done. I think you've missed one thing though. This is not the correct font to use from the Figma design. The Figma uses a combination of `Roboto` and `
Crete Round`. Kindly fix this small change to get your design even closer to the Figma design 🤗